hack/make/manifests/crd.mk

```diff
@@ -1,6 +1,14 @@
 -include config.mk
 -include ../prerequisites.mk
 
+helm/version:
```

Comment:
```suggestion
## Sets the Helm Charts version and appVersion
helm/version:
```

Reply:
Also I meant putting this target in a different file or folder since it has nothing to do with the manifests, not just renaming it